src/web/screens/sharingPermissions.tsx

```diff
@@ -113,6 +115,13 @@ function SharingPermissions() {
     }
   };
 
+  const showPubSharingMessage = () => {
+    const publisher = (participant?.types || []).some((type) => type.typeName === 'Publisher');
+    const sharer = (participant?.apiRoles || []).some((role) => role.roleName === 'SHARER');
+
+    return publisher && !sharer;
```

Comment:
Nitpick: This looks like it could just be a const bool instead of a function.

Reply:
changed to a bool